README.md

```diff
@@ -255,30 +262,31 @@ dictionary XRWebGLLayerSourceInit {
   boolean ignoreDepthValues = false;
 }
 
-interface XRWebGLGraphicsBinding {
-  constructor(XRSession session, WebGLRenderingContext context);
+interface XRWebGLTextureLayerFactory {
+  constructor(XRSession session, XRWebGLRenderingContext context);
 
   double getNativeProjectionScaleFactor();
 
   Promise<XRProjectionLayer> requestProjectionLayer(XRWebGLLayerInit init); // Note different dictionary
-  Promise<XRQuadLayer> requestQuadLayer(XRWebGLLayerSourceInit init);
-  Promise<XRCylinderLayer> requestCylinderLayer(XRWebGLLayerSourceInit init);
-  Promise<XREquirectLayer> requestEquirectLayer(XRWebGLLayerSourceInit init);
-
-  WebGLFramebuffer? getCurrentFramebuffer(XRLayer layer);
+  Promise<XRQuadLayer> requestQuadLayer(XRLayerInit init);
+  Promise<XRCylinderLayer> requestCylinderLayer(XRLayerInit init);
+  Promise<XREquirectLayer> requestEquirectLayer(XRLayerInit init);
+  
+  XRWebGLTextureSubImage? getViewSubImage(XRLayer layer); // for mono layers
+  XRWebGLTextureSubImage? getViewSubImage(XRLayer layer, XRView view); // for stereo layers
 }
 
-interface XRWebGL2GraphicsBinding {
-  constructor(XRSession session, WebGL2RenderingContext context);
+interface XRWebGLFramebufferLayerFactory {
+  constructor(XRSession session, XRWebGLRenderingContext context);
 
   Promise<XRProjectionLayer> requestProjectionLayer(XRWebGLLayerInit init);
-  Promise<XRQuadLayer> requestQuadLayer(XRWebGLLayerSourceInit init);
-  Promise<XRCylinderLayer> requestCylinderLayer(XRWebGLLayerSourceInit init);
-  Promise<XREquirectLayer> requestEquirectLayer(XRWebGLLayerSourceInit init);
-  Promise<XRCubeLayer> requestCubeLayer(XRWebGLLayerSourceInit init); // Note only available with WebGL 2
-
-  WebGLTexture? getCurrentColorTexture(XRLayer layer);
-  WebGLTexture? getCurrentDepthStencilTexture(XRLayer layer);
+  Promise<XRQuadLayer> requestQuadLayer(XRLayerInit init);
+  Promise<XRCylinderLayer> requestCylinderLayer(XRLayerInit init);
+  Promise<XREquirectLayer> requestEquirectLayer(XRLayerInit init);
+  Promise<XRCubeLayer> requestCubeLayer(XRLayerInit init); // Note only available with WebGL 2
```

Comment:
This needs to move to the `XRWebGLTextureLayerFactory`, since you can't represent a cube map with a single framebuffer, but can with a single texture.